amethyst_ui/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "amethyst_ui"
+version = "0.1.0"
+authors = ["Jacob Kiesel <kieseljake@gmail.com>"]
```

Comment:
We'll need some more metadata in order to release it, you wanna do that in a follow-up?

Reply:
I'd rather do it here so I don't forget.  Can you give me suggestions on what kinds of things to add?

Reply:
```toml
description = "Amethyst UI crate"
keywords = ["ui", "specs", "ecs", "amethyst"]
categories = ["game-engines"]

documentation = "https://www.amethyst.rs/doc/develop/doc/amethyst_ui/index.html"
homepage = "https://www.amethyst.rs/"
repository = "https://github.com/amethyst/amethyst"
license = "MIT/Apache-2.0"
```

Reply:
You can get `cargo package`, I think that fails on missing metadata

Reply:
Done.